portfolio/src/main/webapp/script.js

```diff
@@ -48,6 +48,26 @@ function getMessage() {
   });
 }
 
+function getComments() {
+    fetch('/data').then(response => response.json()).then((comments) => { 
+    const commentsListElement = document.getElementById('comments-container');
+    commentsListElement.innerHTML = '';
+    commentsListElement.appendChild(
+        createListElement(comments[0]));
+    commentsListElement.appendChild(
+        createListElement(comments[1]));
+    commentsListElement.appendChild(
+        createListElement(comments[2]));
```

Comment:
Can you re-write this in a way that doesn't assume a specific number of comments?